File: Hahn.ApplicatonProcess.Application/Hahn.ApplicatonProcess.December2020.Data/Applicant.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Hahn.ApplicatonProcess.December2020.Data
{
    public class Applicant
    {
        public int ID { get; set; }
        public string Name { get; set; }
        public string FamilyName { get; set; }
        public string Address { get; set; }
        public string CountryOfOrigin { get; set; }
        public string EMailAdress { get; set; }
        public int Age { get; set; }
        public bool Hired { get; set; }
    }
}

File: Hahn.ApplicatonProcess.Application/Hahn.ApplicatonProcess.December2020.Web/src/models/Applicant.ts
export class Applicant {
  id: number;
  name: string;
  familyName: string;
  address: string;
  countryOfOrigin: string;
  eMailAdress: string;
  age: number;
  hired: boolean;

  public constructor(init?: Partial<Applicant>) {
    Object.assign(this, init);
  }
}
